src/components/v1/V1Project/ProjectActivity/eventElems/TapEventElem.tsx

```diff
@@ -57,6 +60,13 @@ export default function TapEventElem({
 
   if (!event) return null
 
+  const handleProject = (projectId: number, allocator: string) => {
+    if (allocator.toLowerCase() === V1_V3_ALLOCATOR_ADDRESS.toLowerCase()) {
```

Comment:
**suggestion** a slightly more 'correct' approach might be using ethers `getAddress` https://docs.ethers.org/v5/api/utils/address/#utils-getAddress

Reply:
thanks for this tip, didn't know about this utility function!

Reply:
One point of caution is that `getAddress` will throw if the input is invalid (can't remember the exact cases it fails on). Maybe we should add a `safeGetAddress` util...out of scope tho.